examples/python/face_tracking/main.py

```diff
@@ -239,7 +239,6 @@ def detect_and_log(self, image: npt.NDArray[np.uint8], frame_time_nano: int) ->
         )
         rr.log("video/landmarker/faces", rr.Clear(recursive=True))
         rr.log("reconstruction/faces", rr.Clear(recursive=True))
-        rr.log("blendshapes", rr.Clear(recursive=True))
 
         for i, (landmark, blendshapes) in enumerate(
             zip(detection_result.face_landmarks, detection_result.face_blendshapes)
```

Comment:
Those clear happen because the number of face detected can vary from 0 to N. That's why we clear `faces/**` and the re-populate `faces/$i/**`. Maybe a cleaner way to go about this would be to detect when faces disappear and clear only those ones.

Reply:
I don't understand the intent here.
Clearing effectively doesn't affect range queries at all, so what is the expected behavior we should expect? I don't get it.

I.e. this:
```python
for step in range(0, 64):
    rr.set_time_sequence("step", step)
    rr.log("scalar", rr.TimeSeriesScalar(math.sin(step / 10.0)))
```
https://app.rerun.io/pr/4949/index.html?url=https://storage.googleapis.com/rerun-builds/pull_request/4950/scalar_no_clears.rrd

results in the same exact graph as this:
```python
for step in range(0, 64):
    rr.set_time_sequence("step", step)
    rr.log("scalar", rr.TimeSeriesScalar(math.sin(step / 10.0)))
    rr.log("scalar", rr.Clear(recursive=True))
```
https://app.rerun.io/pr/4949/index.html?url=https://storage.googleapis.com/rerun-builds/pull_request/4950/scalar_clears.rrd

Reply:
You're right, not clearing here as no effect, so it's ok to remove it (but I'd leave a TODO to explain why this one is not cleared while the others are).

Incidentally, it would be really nice if those `Clears` would actually translate into breaks in the timeseries to avoid those ugly jumps:
<img width="1026" alt="image" src="https://github.com/rerun-io/rerun/assets/49431240/6c931351-b566-42f0-a9fe-df1ae9a4bdba">

Reply:
Haaa interesting! Let me see if there's something quick we can do about this.